src/main/java/frc/robot/utils/MoPrefs.java

```diff
@@ -13,12 +14,29 @@
 
 public final class MoPrefs {
     public static Pref<Double> maxDriveRpm = doublePref("Drive Max RPM", 6000.0);
+    public static Pref<Double> maxDriveSpeed = doublePref("Drive Max Speed m/s", 6000.0);
+    public static Pref<Double> maxTurnSpeed = doublePref("Turn Max Speed m/s", 6000.0);
     public static Pref<Double> driveDeadzone = doublePref("Drive Deadzone", 0.05);
     public static Pref<Double> driveCurve = doublePref("Drive Curve", 1);
     public static Pref<Double> driveSlowSpeed = doublePref("Drive Slow Speed", 0.5);
     public static Pref<Double> turnSlowSpeed = doublePref("Turn Slow Speed", 0.25);
     public static Pref<Double> driveRampTime = doublePref("Drive Ramp Time", 0.25);
 
+    public static Pref<Double> flZero = doublePref("FL Drive Zero", 0);
+    public static Pref<Double> flScale = doublePref("FL Drive Scale", 1);
+    public static Pref<Double> frZero = doublePref("FR Drive Zero", 0);
+    public static Pref<Double> frScale = doublePref("FR Drive Scale", 1);
+    public static Pref<Double> rlZero = doublePref("RL Drive Zero", 0);
+    public static Pref<Double> rlScale = doublePref("RL Drive Scale", 1);
+    public static Pref<Double> rrZero = doublePref("RR Drive Zero", 0);
+    public static Pref<Double> rrScale = doublePref("RR Drive Scale", 1);
+
+    // Motor Scale: units of ticks per meter
+    public static Pref<Double> flDriveMtrScale = doublePref("FL Drive Motor Scale", 1);
+    public static Pref<Double> frDriveMtrScale = doublePref("FR Drive Motor Scale", 1);
+    public static Pref<Double> rlDriveMtrScale = doublePref("RL Drive Motor Scale", 1);
+    public static Pref<Double> rrDriveMtrScale = doublePref("RR Drive Motor Scale", 1);
```

Comment:
We should probably adjust these names to be more obvious. For example, the difference between flScale and flDriveMtrScale is confusing, and flScale should probably be renamed something like flTurnMtrScale.